.github/workflows/fedora-interop.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM fedora:38
+RUN dnf -y update
+RUN dnf -y install perl
+RUN dnf -y install git tmt-all
+RUN dnf -y remove openssl
+RUN cd / && git clone https://gitlab.com/not4pedro/interop
+RUN cd interop && git checkout 68fb662546d0b1f1fe5cab88218524e7db5764d3
```

Comment:
Doable and probably ok as the test repo is not really large. I can also create a specific branch for openssl if you like.

Other approach, that we used in GnuTLS upstream, was having tests directly in the GnuTLS repo as a git submodule.

Whatever works best for openssl.

Reply:
Yeah, I'm trying to decide what would be best here.  I like the idea of having the interop tests be present in the openssl repo so we have a bit more of a direct path to address fixes in openssl directly, but if these tests are under active development I don't think that really makes sense, as I'd like to incorporate new tests as they are developed.

does the interop repo have plans for tagging stable points or stable branches?  If so, I think the best approach might be to not incorporate the tests in the docker image, but rather clone them at run time in the ci yaml